test/fixtures/rgba-double-fallback.css

```diff
@@ -0,0 +1,7 @@
+.foo {
+  background: rgba(153, 221, 153, 0.8);
+  background: rgba(153, 221, 153, 0.8);
```

Comment:
Perhaps we should remove this line in this case. `decl.prop === dec.prev.().prop &&  decl.value === decl.prev(). value`

But I do not know if this is the plugin role